src/renderer/Components/RecevingPackage/index.jsx

```diff
@@ -27,6 +31,11 @@ function ReceivingPackage() {
     if (!VALID_KEY.includes(event.key) || event.key === " ") {
       event.preventDefault();
     }
+    if (!isNaN(event.key) && event.key.trim() !== "") {
+      const newNumbers = [...numbers];
+      newNumbers[index] = event.key;
+      setNumbers(newNumbers);
+    }
```

Comment:
```suggestion
      const tempInputNumbers = [...inputNumbers];
      tempInputNumbers[index] = event.key;
      setInputNumbers(tempInputNumbers);
    }
```
같은 맥락으로 위에서 제안한 변수명을 사용해 보았습니다 :)